dex/pair/src/fee.rs

```diff
@@ -30,19 +33,17 @@ pub trait FeeModule:
     }
 
     #[endpoint(whitelist)]
-    fn whitelist_endpoint(&self, address: ManagedAddress) -> SCResult<()> {
-        self.require_permissions()?;
+    fn whitelist_endpoint(&self, address: ManagedAddress) {
+        self.require_permissions();
         let is_new = self.whitelist().insert(address);
-        require!(is_new, "ManagedAddress already whitelisted");
-        Ok(())
+        assert!(self, is_new, ERROR_ALREADY_WHITELISTED);
     }
```

Comment:
I'd say simply insert and don't return an error if the user was already whitelisted, as you're not changing anything in storage anyway, so nothing to revert.

Reply:
RV Suggested this behavior is better